packages/node-cli/src/metadata/NodeCommandsMetadata.json

```diff
@@ -4,14 +4,53 @@
         "alias": "sa",
         "description": "Sets up an account to use with the SuiteCloud CLI for Node.js.",
         "isSetupRequired": false,
-        "forceInteractiveMode": true,
```

Comment:
now that we have new commands removal of `forceInteractiveMode` is still needed? what about the options below(`authid`, `accountid`, etc.) ?

Reply:
This is not really a new command.
In the `CommandGenerators.json`, I added a new option for `nonInteractiveGenerator` so both interactive and non-interactive use the same command but with different options and executors.